### DISCUSSION TITLE
Not received the perks yet

### DISCUSSION THREAD

**Discussion by ashutoshdinde** [community/community#141964] [GitHub Education]
### Hi everyone,

I was approved for GitHub educational benefits on 16th Oct, but it’s now been four days, and I haven’t received any confirmation or access. Has anyone else experienced a delay? Any advice on how to resolve this would be appreciated!

**Comment by davevad93** [comment]
Hi @ashutoshdinde ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ashutoshdinde was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
